Gemfile

```diff
@@ -17,6 +17,8 @@ gem 'coffee-rails', '~> 4.1.0'
 gem 'angularjs-rails'
 gem 'angular_rails_csrf'
 
+gem 'webpacker'
```

Comment:
I included this because I was under the misunderstanding that you're running Angular 4 with Rails 4.2.4, and angularjs-rails is for AngularJS/1.  Please disregard it.